bidderPatterns.js

```diff
@@ -300,4 +300,41 @@ var bidderPatterns = {
   'Trustx': [
     'sofia.trustx.org/hb'
   ],
+  'Realvu': [
+    'ac.realvu.net/realvu_boost.js'
```

Comment:
is this the bidding endpoint? Looks suspicious because it ends in `.js`

Reply:
It seems so?  I grabbed it from this line of code in their adapter https://github.com/prebid/Prebid.js/blob/master/modules/realvuBidAdapter.js#L37 and when I attempted to load their test page that was provided in https://github.com/prebid/Prebid.js/pull/1571